frontend/src/workStation/CodeGroup.jsx

```diff
@@ -186,13 +188,31 @@ function CodeGroup() {
           borderRadius={4}
         >
           <Box m={4}>
-            <ul>
-              {allMembers?.length > 0
-                ? allMembers.map((member, index) => {
-                    return <li key={index}>{member.user.fullName}</li>;
-                  })
-                : "No members yet. Add new members"}
-            </ul>
+            {groupData.id ? (
+              <ul>
+                <li
+                  onClick={() =>
+                    handleUserProfileClick(groupData.creator.username)
+                  }
+                >
+                  {groupData.creator.fullName}&nbsp;&nbsp;(creator)
```

Comment:
You may already be planning to do this, but I would use a component that makes it clear that the user names are clickable. In the current setup it just looks like plain text

Reply:
I have updated this to a Button component instead. Thanks